app/lib/stores/workbench.ts

```diff
@@ -476,91 +480,190 @@ export class WorkbenchStore {
     return result;
   }
 
-  async publish(chatId: string, title: string) {
-    this.currentView.set('code');
+  async injectTokenEnvironment(shell: BoltShell, accessToken: string) {
+    try {
+      const wc = await webcontainer;
+      const setupScript = '#!/bin/sh\n\nexport V8_ACCESS_TOKEN="' + accessToken + '"';
+      await wc.fs.writeFile('setup.sh', setupScript);
+      await shell.executeCommand(Date.now().toString(), 'source ./setup.sh');
+      await wc.fs.rm('setup.sh');
```

Comment:
IMHO, just sending `export V8_ACCESS_TOKEN=xxx` to shell without remaining temporary if possible.

if it doesn't work, how about moving `ws.fs.rm()` to `finally` block for safety?

Reply:
I can do that, but I don't want to expose the access token on the user screen.

Reply:
Fair enough. then how about moving `ws.fs.rm()` to finally? it helps when `shell.executeCommand()` throwing.

Reply:
Sure, and I will prevent committing that file as well.